src/metrics/metrics.service.lexi.js

```diff
@@ -26,6 +26,10 @@ angular
             getConsumption: {
                 url: "/metrics/:serviceName/consumption",
                 method: "GET"
+            },
+            setQuota: {
```

Comment:
Since the GET on /metrics/:serviceName is cached, you might want to add the interceptor (see line 20) to reset the cache when calling the PUT on /metrics/:serviceName/quota

Reply:
Nice tips @Netnix ! Just updated the PR